sdk/recoveryservicesbackup/azure-resourcemanager-recoveryservicesbackup/src/test/java/com/azure/resourcemanager/recoveryservicesbackup/RecoveryServicesBackupManagerTests.java

```diff
@@ -0,0 +1,295 @@
+    private String resourceGroupName = "rg" + randomPadding();
+    private RecoveryServicesBackupManager recoveryServicesBackupManager;
+    private RecoveryServicesManager recoveryServicesManager;
+    private ResourceManager resourceManager;
+    private boolean testEnv;
+
+    @Override
+    public void beforeTest() {
+        final TokenCredential credential = new DefaultAzureCredentialBuilder().build();
+        final AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
+
+        recoveryServicesManager = RecoveryServicesManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile);
+
+        recoveryServicesBackupManager = RecoveryServicesBackupManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile);
+
+        resourceManager = ResourceManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile)
+            .withDefaultSubscription();
+
+        // use AZURE_RESOURCE_GROUP_NAME if run in LIVE CI
+        String testResourceGroup = Configuration.getGlobalConfiguration().get("AZURE_RESOURCE_GROUP_NAME");
+        testEnv = !CoreUtils.isNullOrEmpty(testResourceGroup);
+        if (testEnv) {
+            resourceGroupName = testResourceGroup;
+        } else {
+            resourceManager.resourceGroups()
+                .define(resourceGroupName)
+                .withRegion(REGION)
+                .create();
+        }
+    }
+
+    @Override
+    protected void afterTest() {
+        if (!testEnv) {
+            resourceManager.resourceGroups().beginDeleteByName(resourceGroupName);
+        }
+    }
+
+    @Test
+    @DoNotRecord(skipInPlayback = true)
+    public void testCreateProtectionPolicy() {
+        Vault vault = null;
+        ProtectionPolicyResource protectionPolicyResource = null;
+        String randomPadding = randomPadding();
+        try {
+            String vaultName = "vault" + randomPadding;
+            String policyName = "policy" + randomPadding;
+
+            // @embedmeStart
+            List<SubProtectionPolicy> lstSubProtectionPolicy = new ArrayList<>();
+            lstSubProtectionPolicy.add(new SubProtectionPolicy()
+                .withPolicyType(PolicyType.FULL)
+                .withSchedulePolicy(
+                    new SimpleSchedulePolicy()
+                        .withScheduleRunFrequency(ScheduleRunType.WEEKLY)
+                        .withScheduleRunDays(
+                            new ArrayList<DayOfWeek>() {
+                                {
+                                    add(DayOfWeek.SUNDAY);
+                                    add(DayOfWeek.TUESDAY);
+                                }
+                            })
```

Comment:
Same as HashMap, don't do this. Either use `Arrays.asList` or use a `List xxx = new ArrayList(); xxx.add(..);`.

Reply:
Fixed in the new version.